src/banana.js

```diff
@@ -109,7 +109,14 @@ module.exports = function bananaChecker( dir, options, logErr ) {
 
 		for ( index in keys ) {
 			message = keys[ index ];
-			originalParameters = sourceMessages[ message ].match( /\$\d/g );
+			if ( sourceMessages[ message ] === undefined ) {
+				// An unused translation. This happens on commits that remove messages, 
+				// which are typically removed from en.json and qqq.json, letting
+				// translations be removed by a localisation update instead. 
+				originalParameters = null;
```

Comment:
Changed from `undefined` to `null` so that it matches the bottom value of `.match()`.